docs/developer/checkout.mdx

```diff
@@ -767,8 +767,9 @@ As a result, we get an updated checkout object with new checkout email set:
   }
 }
 ```
-
-## Payment
+## Finalizing checkout with Saleor's payment gateways
+The payment flow that uses a Saleor's payment gateway or payment synchronous webhooks to process a payment.
+### Payment
 
```

Comment:
We should avoid mentioning JS, payments can be processed in other platforms than web

Reply:
changed